app/assets/stylesheets/application.tailwind.css

```diff
@@ -2,14 +2,14 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
-@import './stylesheets/buttons.css';
-@import './stylesheets/layout.css';
-@import './stylesheets/typography.css';
-@import './stylesheets/buttons.css';
-@import './stylesheets/lesson-content.css';
-@import './stylesheets/lesson-notes.css';
-@import './stylesheets/prism-theme.css';
-@import './stylesheets/pagy.css';
-@import './stylesheets/animations.css';
-@import './stylesheets/static-pages.css';
-@import './stylesheets/charts.css';
+@import './custom/buttons.css';
```

Comment:
I changed this because we had a `assets/stylesheets/stylesheets/` directory structure. Is there a better directory name than `custom`?

Reply:
Thanks @MaoShizhong, custom_styles sounds better 😎 

I think it would be worth differentiating if we had more files. But with so few, I think the DX is improved slightly having them all in the one spot.